meshdb/meshdb.go

```diff
@@ -45,11 +45,22 @@ func (o Order) ID() []byte {
 	return o.Hash.Bytes()
 }
 
+// Metadata is the database representation of MeshDB instance metadata
+type Metadata struct {
+	EthereumNetworkID *big.Int
```

Comment:
`ETHEREUM_NETWORK_ID` is supplied as an `int` and is unlikely to become so large as to need representation as a `*big.Int`. My vote would be we represent it as an `int` in the DB as well.

Reply:
Totally agree. Meant to make that change in my last commit. I initially used `big.Int` when under the impression the value should be indexed and that type provides a simple way to access a byte slice representation.

Admittedly doesn't make a lot of sense looking back on it 😂